UPGRADING.md

```diff
@@ -19,6 +19,47 @@ Done!
 The following instructions describe modifications to the standard upgrade process required due to
 specific changes. Items are listed in reverse chronological order.
 
+## 4/28/16 - Upgrade Django from 1.7 to 1.9
```

Comment:
Please use three `###` to keep the formatting consistent with the other upgrade headers.